cratera/papers
# Setup

# Description

Add the foundational dependencies we'll need for the project: pnpm, typescript, eslint, prettier, react-native/expo, tailwind, supabase, react-query, expo-router, jest, storybook, etc.

## Caveats

A simple (temporary) auth flow was implemented just to test supabase/react-query.


## How to test

Run `pnpm start` to see the actual app. You can login using the [test account details](https://www.notion.so/jcdvr/Services-we-use-a0204b8d1bd04f278370582d6800bcd0?p=2b17507f75b9456b91fb7d33a7cb8989&pm=s).
Run `pnpm storybook` to see the storybook app.

## Additional Notes

We are still using the default managed expo workflow which allow us to use the [Expo Go](https://expo.dev/client) app.

---

[Notion task](https://www.notion.so/jcdvr/Task-board-caeb522237b14e1599875c72613b1c27?p=987d3a0f277c4bb6973e5d8ab8efa804&pm=s)
